### PR TITLE
make navigate to OtherTabs on select it

### LINKED ISSUE #199: Make the tab selected in the drawer open in OtherScreen
## Kind (Required)

- Improvement

 ## Overview (Required)

- Now when I select a tab from the drawer, it opens the Home tab every time. Please make sure that the tab selected from the drawer is opened in OtherScreen.

 ## Links	

-

### PR DESCRIPTION
**Pull request by ShebangDog**
## Issue
- close #199 

## Overview (Required)
- Change selected item in Drawer on tap OtherTab.

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Review by takahirom — APPROVED** [review]
Looks good!✨
Can you check CI status?

File: uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/other/OtherScreen.kt [hunk 1]
```diff
@@ -185,7 +178,10 @@ private fun AppBar(
 fun PreviewOtherScreen() {
     ConferenceAppFeederTheme {
         OtherScreen(
-            initialSelectedTab = OtherTabs.AboutThisApp,
+            selectedTab = OtherTabs.AboutThisApp,
+            onSelectTab = {
+
```
Comment (punchdrunker) [inline uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/other/OtherScreen.kt:183]: Could you remove this empty line, please?

Comment (ShebangDog) [reply uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/other/OtherScreen.kt:183]: ok! thankyou😆

**Review by takahirom — APPROVED** [review]
Thank you!

**Pull request merged**